### DISCUSSION TITLE
Strapi plugin /w Gatsby v5

### DISCUSSION THREAD

**Discussion by keegankb93** [gatsby-uc/plugins#348] [Q&A] [answered]
Is the Strapi plugin ready for Gatsby v5 or still in-progress?

**Comment by moonmeister** [comment, marked as answer]
The `peerDependencies` have been updated, which we believe is all that was needed. Because this plugin has no tests it's a little hard to know for sure, so please do your own verification. I believe there are a couple of issues with our 3.0.0 release that we're working on in #345

**Reply by keegankb93** [reply]
Great, thank you. Not in a huge rush, so I'll keep an eye out on those fixes. And thank you and @laurenskling for taking this over.

**Reply by laurenskling** [reply]
@keegankb93 you can give [3.0.2](https://www.npmjs.com/package/gatsby-source-strapi/v/3.0.2) a spin. Let us know what it does in Gatsby 5 :)

**Comment by GregPetropoulos** [comment]
How do I get remote url  into childImageSharp if this plugin is not working with Gatsby5? I can not version down due to requirements
- warn Plugin gatsby-plugin-remote-images is not compatible with your gatsby version 5.3.3 - It requires gatsby@^2.2.0 ||
^3.0.0 || ^4.0.0
- https://github.com/gatsbyjs/gatsby/discussions/37437#discussion-4745635

**Reply by laurenskling** [reply]
This is not something we can help with. You could open a issue on gatsby-plugin-remote-images about supporting Gatsby 5

**Reply by moonmeister** [reply]
If that plugin is going unmaintained you could also talk about moving it into this repo with the current maintainers.

**Comment by GregPetropoulos** [comment]
@laurenskling thanks for the quick feedback and direction
